### ISSUE TITLE
Some of the images are blurry

### ISSUE THREAD

**Issue by dphiffer** [eyebeam/eyebeam.org#4]
From Joanna:

> Do you know if there are specific specs for these? Mainly resident images on /residency page and the /event images

I haven't adjusted the image sizes, but maybe I should, maybe bump the large/medium sizes to be larger? And then we can batch resize everything to the new dimensions. Of course if the original upload is not high enough resolution, the new larger sizes won't really help.

Here are the settings as they are set now:
* Thumbnail: 150 x 150 cropped
* Medium: 300 wide, as tall as necessary
* Large: 1024 wide, as tall as necessary

Basically I use `thumbnail` for staff profile images, `medium` for 1/3 width modules, `large` for hero images, 1/2, and full width modules. We can also define additional sizes, if that helps fix specific instances in the layout.

**Comment by leigler** [comment]
If i remember correctly, Joanna had wanted a fixed image proportion for the 1/3 width modules (this mainly has to do with their cropping tool on the backend I suppose?) 
If there's time, I think i'd suggest giving the backend a 3/4 cropping ratio option...

**Comment by dphiffer** [comment]
Would 3/4 be like 300 wide by 400 tall?

**Comment by dphiffer** [comment]
One thing to point out, is if we do go with something like 300x400, a human should always double check each image crop, to make sure we don't crop down the middle of somebody's head or something.

**Comment by leigler** [comment]
agreed— i don't think it should be an automatic crop, but a crop option (like a fixed ratio). 
I think depending on the image there might be a need for the vertical proportion as well as the horizontal proportion.

**Comment by joannagould** [comment]
Makes sense to me.

**Comment by joannagould** [comment]
Update to this - I think the medium and large sizes need to be bumped up. It's happening on the [/people](https://staging.eyebeam.org/people/) and [/event](https://staging.eyebeam.org/events/) pages. For example see the same image three different places below: 

Home page:
<img width="812" alt="screen shot 2018-03-05 at 3 35 04 pm" src="https://user-images.githubusercontent.com/37080849/36998419-df9bad4c-208a-11e8-98f3-23780da39706.png">

Actual event sub-page:
<img width="858" alt="screen shot 2018-03-05 at 3 35 17 pm" src="https://user-images.githubusercontent.com/37080849/36998445-f2fb96a4-208a-11e8-9de1-40224199b844.png">

/Event page (grainy):
<img width="962" alt="screen shot 2018-03-05 at 3 35 25 pm" src="https://user-images.githubusercontent.com/37080849/36998443-f06e012e-208a-11e8-9b9e-7c0803bef802.png">

**Comment by madpenny** [comment]
Hey,

I'm adding a few posts now. It seems that the "featured image" function on the post renders the image high quality. But the other "add image" button (which I think creates the image link for the modular function on the home page?) is the one rendering the image poorly.  
<img width="174" alt="screenshot 2018-03-06 13 22 11" src="https://user-images.githubusercontent.com/31375201/37050368-aca4d0c8-2141-11e8-8e83-4fa80f3f4919.png">
<img width="279" alt="screenshot 2018-03-06 13 22 17" src="https://user-images.githubusercontent.com/31375201/37050369-acb54e8a-2141-11e8-87e9-173e96822348.png">


-Maddie

**Comment by dphiffer** [comment]
Okay I've bumped the widths up to 320, 640, and 1280.

I think it's an improvement. If this looks okay to you all, I can batch resize everything.

<img width="841" alt="screen shot 2018-03-06 at 5 03 50 pm" src="https://user-images.githubusercontent.com/38114/37061163-8bd05f56-2160-11e8-8448-53e7dfbf945c.png">

**Comment by joannagould** [comment]
That looks good. Can we batch crop them too or we should do that on our end and re-load the images? 
Also will this carry over to the /ideas page as well?

**Comment by dphiffer** [comment]
This will apply to _all_ images uploaded to WordPress and we can run it again later if we decide to. Basically the originals can get re-resized as much as we need them to be.

**Comment by dphiffer** [comment]
Oh, but maybe your question is about re-applying custom crop positions? That we probably would need to redo. There's a whole lot more to say about how WordPress does images.

**Comment by dphiffer** [comment]
I just started a [mass resize](https://staging.eyebeam.org/wp-admin/tools.php?page=regenerate-thumbnails). It's 29% done, and has only been running for about 5 minutes.

**Comment by dphiffer** [comment]
Ok, mass resize is done. And... yeah, I think the losing-the-crops thing is a bummer. There's a weird thing that happens with images in WordPress; the maximum sizes don't crop for any size except thumbnails. That is to say, if you specify 300 x 300 for the medium, and then upload an image that's 300 x 400, it won't make a 300 x 300 image, it'll make one that's 225 x 300. Hopefully that makes sense.

However, we can create _new_ sizes that will crop at a specific dimension, it's just not something the built-in medium/large sizes do.

**Comment by dphiffer** [comment]
I’m not sure we are 100% there on image sizes yet, but I think the blurriness should be gone now.
